content/200-concepts/100-components/02-prisma-client/030-crud.mdx

```diff
@@ -777,6 +777,70 @@ To resolve this error, you can:
   const transaction = await prisma.$transaction([deletePosts, deleteUser])
   ```
 
+### Delete all records from all tables
+
+Sometimes you want to remove all data from all tables but keep the actual tables. This can be particularly useful in a development environment.
+
+The following examples show how to remove all data (including relations) from all tables in your database, with `PostgreSQL`, `SQLite` and `MySQL`. The `SQLite` and `MySQL` examples use a `transaction` to batch the queries together so that they are executed on the same connection.
```

Comment:
- the DB names are proper nouns and hence shouldn't be written as inline code (ideally, inline code is used only  when referring to things that "appear like this in code", e.g. variable names, class names, model names, function names, strings, ...)
- let's link the mention of `$transaction` to where it's explained in the docs 

```suggestion
The following examples show how to remove all data (including relations) from all tables in your database, with PostgreSQL, SQLite and MySQL. The SQLite and MySQL examples use a [`$transaction`](TODO) to batch the queries together so that they are executed on the same connection.
```

Reply:
Argh, sorry, yeah. Ive got to remember this